README.md

```diff
@@ -0,0 +1,50 @@
+# CA - Workflow - Noroff
+
+[![Automated Unit Testing](https://github.com/pnordboj/social-media-client/actions/workflows/main.yml/badge.svg)](https://github.com/pnordboj/social-media-client/actions/workflows/main.yml)
+
+[![Automated E2E Testing](https://github.com/pnordboj/social-media-client/actions/workflows/e2e-test.yml/badge.svg?branch=workflow)](https://github.com/pnordboj/social-media-client/actions/workflows/e2e-test.yml)
+
+[![pages-build-deployment](https://github.com/pnordboj/social-media-client/actions/workflows/pages/pages-build-deployment/badge.svg)](https://github.com/pnordboj/social-media-client/actions/workflows/pages/pages-build-deployment)
+

```

Comment:
Well done on adding the badges. However, you should add the new configurations information such as how to setup the project on local, how to execute the scripts etc.